website/thunderbird/115.0/whatsnew/index.html

```diff
@@ -0,0 +1,289 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% set active_page = "whatsnew" %}
+{% set hide_header = True %}
+{% extends "includes/base-resp.html" %}
+
+{% block page_css %}<link href="/media/css/115release.css" rel="stylesheet" type="text/css" />{% endblock %}
+
+{% block page_title %}{{ _('What’s New in Thunderbird 115') }}{% endblock %}
+
+{% block site_header %}
+  <header>
+    <div class="container">
+      <h1>
+        {{_('What’s New in
+        <br>
+        <span class="tb-title">Thunderbird</span>
+        <span class="version">115</span>')}}
+      </h1>
+      <img class="supernova" src="/media/img/thunderbird/whatsnew/115/supernova.png" alt="">
+    </div>
+  </header>
+{% endblock %}
+
+{% block content %}
+  <section class="whats-new-section">
+    <figure class="container slider-box">
+      <div id="before"></div>
+      <div id="after"></div>
+      <input type="range" min="1" max="100" value="50" class="slider" name='slider' id="slider" />
```

Comment:
This slider is cool(nice work!). When I click on it to move the bar around, it seems like it holds focus, which causes mouse scrollwheel to scroll the bar around instead of the page, until you click somewhere else. That disrupts looking at the rest of the page after interacting with the slider.

Reply:
What browser / OS? I have tried this on Firefox, Safari, and Chrome on MacOS and I cannot reproduce this. If I click into the slider and use arrow keys it affects the slider but the scrollwheel only scrolls the page for me.

I thought maybe doing a `preventDefault()` onWheel might solve it but that makes it where I can't scroll at all when it's focused.

Reply:
Firefox on Windows has this problem for me, even in Troubleshoot Mode. Chrome does not.